support/mock/mockrt.go

```diff
@@ -297,6 +315,41 @@ func (rt *Runtime) Store() adt.Store {
 
 ///// Mocking facilities /////
 
+type message struct {
+	// message values
+	to     addr.Address
+	method abi.MethodNum
+	params abi.MethodParams
+	value  abi.TokenAmount
+
+	// returns from applying message
+	sendReturn runtime.SendReturn
+	exitCode   exitcode.ExitCode
+}
+
+func (m *message) Key() string {
+	return fmt.Sprintf("to: %v from: %v params: %v value: %v", m.to.String(), m.method, m.params, m.value)
+}
+
+func (rt *Runtime) ExpectSend(toAddr addr.Address, methodNum abi.MethodNum, params abi.MethodParams, value abi.TokenAmount, sendReturn runtime.SendReturn, exitCode exitcode.ExitCode) {
+	if rt.sendingMessage != nil {
+		rt.t.Errorf("previous unsent message %v", rt.sendingMessage.Key())
+	}
```

Comment:
With a slice, this is fine. An actor must be able to send more than one message.